addon_updater.py

```diff
@@ -76,7 +76,9 @@ def __init__(self):
 		self._tag_latest = None
 		self._tag_names = []
 		self._latest_release = None
-		self._include_master = True
+		self._include_branches = False
+		self._include_branch_list = ['master']
```

Comment:
So these two lines seem contradictory include_branches is is false but there's a list of branches to include?